examples/mpi/mpi.acb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+echo $PATH
+if [ "$EUID" -ne 0 ]; then
+    echo "This script uses functionality which requires root privileges"
+    exit 1
+fi
+
+# Start the build with an empty ACI
+acbuild --debug begin
+
+# In the event of the script exiting, end the build
+acbuildEnd() {
+    export EXIT=$?
+    acbuild --debug end && exit $EXIT
+}
+trap acbuildEnd EXIT
+
+# Name the ACI
+acbuild --debug set-name mpi
+
+# Based on ubuntu
+acbuild --debug dep add quay.io/sameersbn/ubuntu    
+
+#Copies the required directory
+
+#A shell script to compile the OSU and SMB and run a sample MPI program
+acbuild copy mpi.sh /mpi.sh
+
+# Install mpi and other libraries required to build hpl
+acbuild --debug run apt-get update
+acbuild --debug run apt-get install libopenmpi-dev openmpi-bin openmpi-doc libatlas-base-dev gcc openssh-server openssh-client make net-tools g++ vim bzip2 unzip
+
+#Download the OSU and SMB benchmarks
+acbuild --debug run wget http://mvapich.cse.ohio-state.edu/download/mvapich/osu-micro-benchmarks-5.3.2.tar.gz
+acbuild --debug run wget http://www.cs.sandia.gov/smb/download/smb.tgz
```

Comment:
I suggest you to introduce some kind of integrity/origin checks (ie. checksum / signature), otherwise there is no way to ensure you are actually getting pristine files.